Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.6
+
+# Set up code directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install Linux dependencies
+RUN apt-get update && apt-get install -y libssl-dev
+
+# Copy over requirements
+COPY setup.py .
+COPY README.md .
+COPY requirements-dev.txt .
+COPY requirements-docs.txt .
```

Comment:
Probably don't need to copy in requirements-docs.txt here, unless you're planning on building the documentation inside this container.